utils/s2n_safety.h

```diff
@@ -97,6 +97,8 @@ static inline void* trace_memcpy_check(void *restrict to, const void *restrict f
 #define S2N_IN_INTEG_TEST ( getenv("S2N_INTEG_TEST") != NULL )
 #define S2N_IN_TEST ( S2N_IN_UNIT_TEST || S2N_IN_INTEG_TEST )
 
+#define S2N_TLS13_TEST_FEATURES (S2N_IN_TEST && getenv("S2N_ENABLE_TLS13_FOR_TESTING"))
```

Comment:
I would prefer to avoid calling this getenv() repeatedly and just store the values once during s2n_init. getenv implementations do a linear search over all environment variables.

Reply:
getenv() in linux should be implemented with a hashtable, but +1 for setting it once in s2n_init. also i would prefer to call it `S2N_TLS13_ENABLED`

Reply:
raycoll is correct that [getenv() implementations](https://github.com/bminor/glibc/blob/master/stdlib/getenv.c) loops through values, which is more important not to look up multiple times.